http-server/src/main/java/io/micronaut/http/server/HttpServerConfiguration.java

```diff
@@ -282,15 +273,6 @@ public void setHost(String host) {
         }
     }
 
-    /**
-     * Sets the default read timeout.
-     *
-     * @param readTimeout The read timeout
-     */
-    public void setReadTimeout(Integer readTimeout) {
-        this.readTimeout = readTimeout;
-    }
-
```

Comment:
This is a breaking change and cannot be done in a 1.x release

Reply:
Reverted changes.